apps/web/styles/globals.css

```diff
@@ -438,3 +438,14 @@ hr {
     opacity: 1;
   }
 }
+
+/* Hide scrollbar for Chrome, Safari and Opera */
+.no-scrollbar::-webkit-scrollbar {
+  display: none;
+}
+
+/* Hide scrollbar for IE, Edge and Firefox */
+.no-scrollbar {
+  -ms-overflow-style: none;  /* IE and Edge */
+  scrollbar-width: none;  /* Firefox */
+}
```

Comment:
These changes are needed to hide the scrollbar for the categories list in the Apps page

Reply:
We already [have this](https://github.com/calcom/cal.com/blob/fef006719fe08c25d7058a462c8e0e609ba5e397/apps/web/styles/globals.css#L73-L90)

Reply:
Good catch, removed!